### DISCUSSION TITLE
How to make a text of this format? PLEASE HELP

### DISCUSSION THREAD

**Discussion by vavilon224** [fabricjs/fabric.js#8912] [Q&A] [answered]
I don't understand how to make text VALLEY Curve etc . Tell
![text](https://github.com/fabricjs/fabric.js/assets/87662684/b47b2033-d52e-4546-9984-253806abf21a)

**Comment by vavilon224** [comment]
@ShaMan123 @melchiar PLease Need HELP

**Reply by melchiar** [reply, marked as answer]
I remember an old stack overflow answer for creating text effects like that. It was built for an old version of fabric, but should be enough to get you started.

https://stackoverflow.com/a/19553200

**Reply by ShaMan123** [reply]
Nice
Using a cache to render as image with transform - clever

**Reply by vavilon224** [reply]
@melchiar  @ShaMan123 Thank you very much) So it's not possible to collect such a text)

**Reply by ShaMan123** [reply]
I didn't reach that conclusion from what @melchiar suggested.
You should add logic to cache drawing.

**Comment by ShaMan123** [comment]
I saw an open discussion about it
I have no idea how it is achieved
